includes/gateways/stripe/includes/give-stripe-helpers.php

```diff
@@ -1270,3 +1270,100 @@ function give_stripe_get_becs_icon_style( $form_id ) {
+	$account_configured_no_ssl_message     = __( 'Credit Card fields are disabled because your site is not running securely over HTTPS.', 'give' );
+
+	if ( 'stripe_becs' === $gateway_id ) {
+		$account_not_configured_no_ssl_message = __( 'Mandate form fields are disabled because Stripe is not connected and your site is not running securely over HTTPS.', 'give' );
+		$account_not_configured_message        = __( 'Mandate form fields are disabled. Please connect and configure your Stripe account to accept donations.', 'give' );
+		$account_configured_no_ssl_message     = __( 'Mandate form fields are disabled because your site is not running securely over HTTPS.', 'give' );
+	} else if ( 'stripe_sepa' === $gateway_id ) {
+		$account_not_configured_no_ssl_message = __( 'IBAN fields are disabled because Stripe is not connected and your site is not running securely over HTTPS.', 'give' );
+		$account_not_configured_message        = __( 'IBAN fields are disabled. Please connect and configure your Stripe account to accept donations.', 'give' );
+		$account_configured_no_ssl_message     = __( 'IBAN fields are disabled because your site is not running securely over HTTPS.', 'give' );
+	} else if ( 'stripe_google_pay' === $gateway_id ) {
+		$account_not_configured_no_ssl_message = __( 'Google Pay button is disabled because Stripe is not connected and your site is not running securely over HTTPS.', 'give' );
+		$account_not_configured_message        = __( 'Google Pay button is disabled. Please connect and configure your Stripe account to accept donations.', 'give' );
+		$account_configured_no_ssl_message     = __( 'Google Pay button is disabled because your site is not running securely over HTTPS.', 'give' );
+	} else if ( 'stripe_apple_pay' === $gateway_id ) {
+		$account_not_configured_no_ssl_message = __( 'Apple Pay button is disabled because Stripe is not connected and your site is not running securely over HTTPS.', 'give' );
+		$account_not_configured_message        = __( 'Apple Pay button is disabled. Please connect and configure your Stripe account to accept donations.', 'give' );
+		$account_configured_no_ssl_message     = __( 'Apple Pay button is disabled because your site is not running securely over HTTPS.', 'give' );
+	}
+
+	$status = true;
+
+	if (
+		! give_stripe_is_account_configured() &&
+		! is_ssl() &&
+		! give_is_test_mode()
+	) {
+		// Account not configured, No SSL scenario.
+		Give()->notices->print_frontend_notice(
+			sprintf(
+				'<strong>%1$s</strong> %2$s',
+				__( 'Notice:', 'give' ),
+				$account_not_configured_no_ssl_message
+			)
+		);
+		$status = false;
+	} elseif ( ! give_stripe_is_account_configured() ) {
+		// Account not configured scenario.
+		Give()->notices->print_frontend_notice(
+			sprintf(
+				'<strong>%1$s</strong> %2$s',
+				__( 'Notice:', 'give' ),
+				$account_not_configured_message
+			)
+		);
+		$status = false;
+	} elseif (
```

Comment:
This should just be `else`, rather than doing the checks that have been previously done in the `if`s.

Reply:
@bradp This will not be `else` scenario. `else` scenario will be when this function returns `true`. So, you can check the use of the function in the Give core or Give Stripe addon to know more about its usage.